README.md

```diff
@@ -1,33 +1,31 @@
-# Operate First template for repositories
+# Project Meteor
 
-Derive new repositories from this template
+Project Meteor is an AICoE team effort to build an end-end Data Science product showcasing the usefulness of Thoth and AI Ops capabilities along with the integration of  Open Data Hub/Red Hat OpenShift Data Science.
 
-List of featurese:
+Similar to how a meteoroid entering the atmosphere, becomes visible as a meteor and finally hitting the Earth's surface as a meteorite, we aim to create better visibility and impact of our team's work through project *Meteor*.
 
-## License
+## Product
 
-This template ensures new repos are created compliant with [ADR 0001](https://www.operate-first.cloud/blueprints/blueprint/docs/adr/0001-use-gpl3-as-license.md) and use GNU GPL v3 license.
+The product is an interactive [Elyra tutorial](https://github.com/AICoE/elyra-aidevsecops-tutorial) web application that is rendered as a [JupyterBook](https://jupyterbook.org/) environment by taking as an input the user's GitHub repository (user's fork of the Elyra tutorial repository) and runs the build/deployment pipelines in the backend to render the notebook environment as the final output. You can also take a look at the product flowchart for more details [here](https://github.com/AICoE/meteor/blob/main/docs/project-flowchart.md).
 
-## AI-CoE CI Github application
+### Elyra Tutorial
 
-AI-CoE CI provides easy and quick integration for build pipelines and checks for pull requests.
+The Elyra tutorial is used to discuss the interface between Data Science and Dev/DevOps using project templates, pipelines and bots. Moreover, it wants to highlight that Data Scientists are not so different from developers and DevSecOps practices and tools can be applied to MLOps ones.
 
-An empty [`.aicoe-ci.yaml`](.aicoe-ci.yaml) is created here, disabling all checks via this CI provider by default. Documentation can be found [here](https://github.com/AICoE/aicoe-ci/).
+The demo application used in this tutorial is the "hello world" for AI: MNIST Classification. To learn more about Elyra, you can check out their [repository](https://github.com/elyra-ai/elyra).
 
```

Comment:
maybe its worth mentioning that the ML app itself is not in focus, its the workflow, the service, the product that we want to show off.